stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -3047,6 +3050,122 @@ impl NakamotoChainState {
         Ok((epoch_receipt, clarity_commit))
     }
 
+    /// Create a StackerDB config for the .miners contract.
+    /// It has two slots -- one for the past two sortition winners.
+    pub fn make_miners_stackerdb_config(
+        sortdb: &SortitionDB,
+    ) -> Result<StackerDBConfig, ChainstateError> {
+        let tip = SortitionDB::get_canonical_burn_chain_tip(sortdb.conn())?;
```

Comment:
Can this be an argument, instead? This is called from the network main loop, so ideally we don't do any gratuitous I/O.

Reply:
Can do!